sdk/identity/identity/test/integration/node/azureKubernetesTest.spec.ts

```diff
@@ -15,10 +15,10 @@ describe("Azure Kubernetes Integration test", function () {
     const subscriptionId = requireEnvVar("IDENTITY_SUBSCRIPTION_ID");
     const podName = requireEnvVar("IDENTITY_AKS_POD_NAME");
 
-    if (process.env.IDENTITY_CLIENT_SECRET) {
+    if (process.env.ARM_OIDC_TOKEN) {
       // Log in as service principal in CI
-      const clientId = requireEnvVar("ARM_CLIENT_ID");
-      const tenantId = requireEnvVar("ARM_TENANT_ID");
+      const clientId = requireEnvVar("AZURE_CLIENT_ID");
+      const tenantId = requireEnvVar("AZURE_TENANT_ID");
```

Comment:
Should these still be IDENTITY_* since it's getting set as a bicep output now?

Reply:
umm, but the users would likely set it as AZURE_CLIENT_ID etc right?

Reply:
We probably have both available - https://github.com/Azure/azure-sdk-for-js/blob/74e3d27debae13fbecab198898f298653da9ba20/sdk/identity/identity/tests.yml#L26 @benbp

Reply:
@benbp The env vars in `tests.yml` can be removed but I have to test more to see if removing them would break any live tests. I'll put this on hold for another PR to clean up the env vars later